go-cosmwasm/types/types_test.go

```diff
@@ -1,27 +0,0 @@
-package types
-
-import (
-	"bytes"
-	"encoding/json"
-	"testing"
-)
-
-func TestCanoncialAddress(t *testing.T) {
-	addr := CanonicalAddress([]byte{17, 121, 0, 93})
-	bz, err := json.Marshal(addr)
-	if err != nil {
-		t.Error(err)
-	}
-	if string(bz) != "[17,121,0,93]" {
-		t.Errorf("Unexpected encoding %s", string(bz))
-	}
-
```

Comment:
I'd say don't remove this file, because if they change it upstream it's just going to give us conflicts.